### DISCUSSION TITLE
修改代码块的高亮样式

### DISCUSSION THREAD

**Discussion by nonlinearthink** [CaiJimmy/hugo-theme-stack#912] [Q&A]
我使用了 [CaiJimmy/hugo-theme-stack-starter](https://github.com/CaiJimmy/hugo-theme-stack-starter) 的模板，然后尝试修改代码块的高亮样式，这样子修改：

```
[highlight]
noClasses = true
...
style = 'onedark'
```

然后，这是渲染出来的结果：

![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/44293628/69dda821-ebcd-4d6f-b742-6e9da7a8a2d1)

是的，代码的样式确实变了，但是有什么办法可以消除外面的那一圈黑色的背景吗？

或者说有什么其他的修改办法？因为我看到 Hugo 的文档里说 `noClasses = false` 好像还需要一个样式表，我还没搞清楚怎么操作。

参考资料：
https://gohugo.io/getting-started/configuration-markup/#highlight
https://gohugo.io/content-management/syntax-highlighting/
https://xyproto.github.io/splash/docs/all.html

**Comment by nonlinearthink** [comment]
我还发现这样子修改之后，超出范围的代码滚动条也不见了

**Comment by Lovirr** [comment]
是这个元素
![image](https://github.com/CaiJimmy/hugo-theme-stack/assets/106920260/b01fe29c-f795-4248-ae4f-9d656d85304f)

直接在custom.scss中加入
```
.article-content .highlight {
  background-color: 你想要的颜色;
}
```
